src/Modals/ModalMessage.tsx

```diff
@@ -27,13 +31,17 @@ const ModalMessage: FunctionComponent<ModalMessageProps> = (props)
     id,
     children,
     variant,
+    role,
+    ariaLive,
   } = props;
   const theme = useContext(ThemeContext);
   return (
     <StyledModalMessage
```

Comment:
This is also not something to fix here, but one downside of using `FunctionComponent`s that wrap `StyledComponent`s is that we lose the implicit props that would already exist on the `StyledComponent`, including things like `role` and `aria-live`. So if this were just doing `export default StyledModalMessage;`, we wouldn't need to specify every html prop that we wanted to use (or things like `ref` values).

Again, not something that needs to change here, but in the future we may end refactoring a few of these to give us more flexibility.